azure-pipelines-dependency-graph-testing.yml

```diff
@@ -1,14 +1,19 @@
-# This pipeline is meant to run the GE test suite with an experimental test runner strategy.
-# The significant changes between this YAML and the primary azure-pipelines.yml file are:
-#  - Utilizes a custom script to filter the test files selected and passed on to pytest
-#  - Removes stages that aren't relevant to testing this strategy (i.e. usage stats)
-#  - Only tests Python 3.8 to improve performance
+# This file is responsible for configuring the `dependency_graph` pipeline (https://dev.azure.com/great-expectations/great_expectations/_build)
+#
+# The pipeline is run under the following conditions:
+#   - On the develop branch whenever a commit is made to an open PR
+#
+# `dependency_graph` runs a subset of test suite, aiming to only test what's relevant to attain coverage over a given set of changes.
+# The pipeline aims to balance both performance and safety to improve the developer experience but is innately less thorough than `great_expectations`.
+#
+# Note that `dependency_graph` makes heavy use of dgtest (https://github.com/superconductive/dgtest)
 
 trigger:
   branches:
     include:
-    - pre_pr-*
+    - pre_pr-* # Can be used to test both `great_expectations` and `dependency_graph` pipelines
```

Comment:
Not sure if people use this prefix but keeping it in case.

If you want to revert to old behavior, a `pre_pr-` branch will do the job.